packages/fxa-settings/src/components/Modal/en-US.ftl

```diff
@@ -0,0 +1,4 @@
+## Modal
+
+modal-close-title = Close modal
```

Comment:
I wonder if we can do better than "Close modal"? Do most folks know what "modal" means?